homeassistant/components/vesync/light.py

```diff
@@ -33,17 +35,20 @@ async def async_discover(devices):
     hass.data[DOMAIN][VS_DISPATCHERS].append(disp)
 
     _async_setup_entities(hass.data[DOMAIN][VS_LIGHTS], async_add_entities)
+    return True
 
 
 @callback
 def _async_setup_entities(devices, async_add_entities):
     """Check if device is online and add entity."""
     entities = []
     for dev in devices:
-        if DEV_TYPE_TO_HA.get(dev.device_type) == "light":
+        if DEV_TYPE_TO_HA.get(dev.device_type) == "dimmerswitch":
             entities.append(VeSyncDimmerHA(dev))
+        elif DEV_TYPE_TO_HA.get(dev.device_type) == "bulb":
+            entities.append(VeSyncBulbHA(dev))
         else:
-            _LOGGER.debug(
+            _LOGGER.warning(
```

Comment:
Keep this at debug level as this will generate issues that we won't be able to do anything about